agent/agent.go

```diff
@@ -118,6 +137,23 @@ func (a *Agent) Name() string {
 	return "agent"
 }
 
+// addPolicy is a placeholder. TODO
```

Comment:
I assume this is a place holder for when the policy-agent functionality is moved into the agent?